app/src/main/java/com/github/se/eduverse/ui/navigation/NavigationActions.kt

```diff
@@ -50,7 +51,7 @@ val LIST_TOP_LEVEL_DESTINATION =
         TopLevelDestinations.CAMERA,
         TopLevelDestinations.OTHERS)
 
-class NavigationActions(
+open class NavigationActions(
```

Comment:
This is to allow mocking the class in the tests

Reply:
yes no problem